src/test/java/nextstep/subway/line/application/LineServiceTest.java

```diff
@@ -0,0 +1,96 @@
+package nextstep.subway.line.application;
+
+import nextstep.subway.line.domain.Line;
+import nextstep.subway.line.domain.LineRepository;
+import nextstep.subway.line.dto.LineRequest;
+import nextstep.subway.line.dto.LineResponse;
+import nextstep.subway.station.application.StationService;
+import nextstep.subway.station.dto.StationResponse;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.stereotype.Service;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+
+import static nextstep.subway.line.domain.LineFixture.신이호선;
+import static nextstep.subway.line.domain.LineFixture.이호선;
+import static nextstep.subway.station.StationFixture.당산역;
+import static nextstep.subway.station.StationFixture.합정역;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+@Service
+public class LineServiceTest {
+
+    LineRepository lineRepository = mock(LineRepository.class);
+    StationService stationService = mock(StationService.class);
+    LineService lineService = new LineService(lineRepository, stationService);
```

Comment:
```suggestion
    @Mock
    LineRepository lineRepository;
    @Mock
    StationService stationService;
    @InjectMocks
    LineService lineService;
```

이렇게 간편하게 사용하실 수 있습니다.

Reply:
코드가 훨씬 가독성이 좋아지네요!
수정해서 반영했습니다. 😁😎